dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report.controller.ts

```diff
@@ -1,26 +1,26 @@
 import { Response } from 'express';
-import { getUtilisationReportDetailsById } from '../../../repositories/utilisation-reports-repo';
+import { UtilisationReportRepo } from '../../../repositories/utilisation-reports-repo';
 import { CustomExpressRequest } from '../../../types/custom-express-request';
 
 type GetUtilisationReportByIdRequest = CustomExpressRequest<{
   params: {
-    _id: string;
+    id: string;
   };
 }>;
 
 export const getUtilisationReportById = async (req: GetUtilisationReportByIdRequest, res: Response) => {
-  const { _id } = req.params;
+  const { id } = req.params;
 
   try {
-    const utilisationReport = await getUtilisationReportDetailsById(_id);
+    const utilisationReport = await UtilisationReportRepo.findOneBy({id: Number(id)});
```

Comment:
Hmm I left as is because I felt this is the sort of basic query we can use the underlying repo for (note if I didn't need to cast to Number it would be simply `UtilisationReportRepo.findOneBy({id});`) so I felt it would be adding more complexity than removing if we create a method and perhaps setting precedence for a pattern that doesn't fully utilise the ORM?